plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/AbstractOpenshiftUIElement.java

```diff
@@ -59,7 +59,8 @@ public boolean equals(Object o) {
 			return true;
 		}
 		AbstractOpenshiftUIElement<?, ?> other= (AbstractOpenshiftUIElement<?, ?>) o;
-		return wrapped.equals(other.getWrapped()) && getParent().equals(other.getParent());
+		return wrapped.equals(other.getWrapped())
+				&& (getParent() == null && other.getParent() == null || getParent().equals(other.getParent()));

```

Comment:
if getParent() == null but other.getParent() != null, then you'll get an NPE calling getParent().equals(...)